File: Class_member/2.rb
class Cs
  def initialize()
  end
  def Cs.class_method()
    p "Class method"
  end
  def instance_method()
    p "Instance_method"
  end
end



i = Cs.new()
Cs.class_method()
# Cs.instance_method()
# i.class_method()
i.instance_method()

File: Class_member/2.py
class Cs:
    @classmethod
    def class_method(cls):
        print("Class Method")
    @staticmethod
    def static_method():
        print("Static Method")
    def instance_method(self):
        print("Instance Method")

i = Cs()
i.instance_method()
Cs.static_method()
Cs.class_method()
